runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -196,6 +214,22 @@ public void onInteractingChanged(InteractingChanged event)
 		}
 	}
 
+	private boolean checkFishOnTiles(NPC fish, Set<WorldPoint> adjacentTiles)
```

Comment:
This can be inlined into `checkFishNextToNet` since it is so short. See my comment on that method